src/include/btmem.h

```diff
@@ -857,29 +857,6 @@ struct __wt_ref {
 #define	WT_REF_SPLIT	 7		/* Parent page split (WT_REF dead) */
 	volatile uint32_t state;	/* Page state */
 
-#ifdef HAVE_DIAGNOSTIC
```

Comment:
@bvpvamsikrishna you should take a look at the changes in btmem.h from this lint PR.